### DISCUSSION TITLE
Renovate skips AutoMergeSchuling if "updateNotScheduled": false is set

### DISCUSSION THREAD

**Discussion by yunex-ts** [renovatebot/renovate#29041] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

## Mode
Self Hosted

## Version
`renovate/renovate:37.353.1`

## Issue
I'm running renovate for multiple Clusters in a wave concept. Therefore it is necessary to create MR on thursday for all systems, for the upcoming week and perform automerge schduling for minor+patch on Monday, Tuesday and Wednesday. That means that it is really important to NOT update existing MR during the week. Only on scheduled day on thursday and also there create new ones in case of skip wave conept.
To do so I configured properly a schedule on thursday and the automerging on Monday, Tuesday and Wednesday.
Noticed that during the week existing MR were updated. So I used the parameter ``"updateNotScheduled": false`` and thought that it will noit update existing MR outside the schedule on thursday.
But while checking logs for scheduling it just quits the checks for an MR...also skips the automergescheduling.

I think that is a bug. That automergingschedule should be checked, if this param is set or not.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment, marked as answer]
- https://github.com/renovatebot/renovate/issues/24275
